pages/api.js

```diff
@@ -87,40 +161,51 @@ const API = ({ api, services }) => {
         rate_limiting={rate_limiting_resume}
       />
 
-      <Menu detail={detail} />
-
       <div id="description" className="ui container">
-        <Content content={content} />
+        <div className="ui equal width grid padded">
+          <div className="four wide column computer only">
+            <Menu
+              detail={detail}
```

Comment:
note for future reference: detail object contains many information not useful for the menu

Reply:
Ok dully noted. I think there will be a major cleanup in api as some stuff are no longer useful